RUN_reduced_form_graphs.R

```diff
@@ -745,119 +917,6 @@ for (i in seq(2,3,1)){
 }
 
 
-# 7. IMR
-# =================================================================
-
-yearly_folder <- "regs_plots_trend/imr/"
-
-var_map <-  rbind(cbind('tx_mi','Infant Mortality Rate'),
-                  cbind('tx_mi_icsap','Infant Mortality Rate - APC'),
-                  cbind('tx_mi_nicsap','Infant Mortality Rate - non-APC'),
-                  cbind('tx_mi_infec','Infant Mortality Rate - Infectious'),
-                  cbind('tx_mi_resp','Infant Mortality Rate - Respiratory'),
-                  cbind('tx_mi_perinat','Infant Mortality Rate - Perinatal'),
-                  cbind('tx_mi_cong','Infant Mortality Rate - Congenital'),
-                  cbind('tx_mi_ext','Infant Mortality Rate - External'),
-                  cbind('tx_mi_nut','Infant Mortality Rate - Nutritional'),
-                  cbind('tx_mi_out','Infant Mortality Rate - Other'),
-                  cbind('tx_mi_illdef','Infant Mortality Rate - Ill-Defined'),
-                  cbind('tx_mi_fet','Infant Mortality Rate - Fetal'),
-                  cbind('tx_mi_24h','Infant Mortality Rate - Within 24h'),
-                  cbind('tx_mi_27d','Infant Mortality Rate - 1 to 27 days'),
-                  cbind('tx_mi_ano','Infant Mortality Rate - 27 days to 1 year'),
-                  cbind('tx_mm',"Maternal Mortality Rate"))
-
-# continuous
-
-for (i in seq(1,3,1)){
```

Comment:
Don't you think it would be good to keep the original IMR "# continuous" loops and adding the robustness routine instead of overwriting it?

Reply:
Hey Michel, we could!  The only reason I didn't was that I left spec=3 as the final graph exported in each group so that we export the graph we need each time.  But if you prefer for simplicity to leave separate, this is fine for me!  Should I edit, or do you?

Reply:
Hi Damian! I did not noticed before that the axis inputs for the robustness plots are the same as the inputs used for outcomes 1,2, and 3. I was worried we were missing the exact inputs that generate the main IMR figure. No need to keep it.